fieldservice_logbook/models/fsm_order.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class FSMOrder(models.Model):
+    _name = "fsm.order"
+    _inherit = ["fsm.order", "fsm.equipment.logbook.mixin"]
+
+    def _read_group_logbook_data(self, logbook):
+        return logbook.read_group(
+            [
+                "|",
+                ("location_id", "in", self.ids),
+                ("equipment_id.location_id", "in", self.ids),
+            ],
+            ["location_id"],
+            ["location_id"],
+        )
```

Comment:
I don't know well those models, but it looks strange to have exactly the same code than for  `fsm.location`
Is it normal to compare a `location_id` with `self`, which isn't a `fsm.location` ?

Reply:
yep, my bad, it should be of course `fsm.order`::`location_id` and `fsm.order`::`equipment_id.location_id`

Reply:
`fsm.order` logs are searched in database by `location_id` and `type=order`